src/common/setup.rs

```diff
@@ -21,37 +21,6 @@ macro_rules! asset_map {
     }
 }
 
-pub mod install {
```

Comment:
This is actually [still used](https://github.com/zellij-org/zellij/blob/a24c7f79f1ac40f1e8554aa1615b0a7e020c7ca5/src/server/mod.rs#L190), but good catch! 
I guess the CI should check with --all_features and maybe also just the default features?

Or we need to address this issue in the makefile.

Reply:
Yep, I guess there's room for improvement with the testsuite then :laughing:

Want me to remove this patch from the patchset?

Reply:
Yes, please do so.